staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -554,17 +554,17 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 			versionedObject = defaultVersionedObject
 		}
 		reqScope.Namer = action.Namer
-		namespaced := ""
+		namespaced := "Cluster"
```

Comment:
Cool. I'll change it back. Are you also fine with the scope label for the metric taking values "Namespace" and "" then?